engine/source/gui/core/guiCanvas.cpp

```diff
@@ -573,14 +573,13 @@ bool GuiCanvas::processInputEvent(const InputEvent* event)
     else if (event->deviceType == XInputDeviceType)
     {
         // Do NOT PUT THE gLastEventTime change here, it will pick up the analog stick noise
-
         //copy the modifier into the new event
         mLastEvent.modifier = event->modifier;
-        
         bool retval = false;
 
         if (event->objType == SI_BUTTON || event->objType == SI_POV)
         {
+            Con::printf("Type was SI_BUTTON etc");
```

Comment:
We shouldn't have print outs in input code, we don't need console spam except for debugging.

Reply:
Yeah, already spotted and removed! Left it in by mistake